patients/add_edit_event_user.php

```diff
@@ -52,7 +52,7 @@
 
  // Exit if the modify calendar for portal flag is not set
 if (!($GLOBALS['portal_onsite_appt_modify'])) {
-    echo htmlspecialchars(xl('You are not authorized to schedule appointments.'), ENT_NOQUOTES);
+    echo htmlspecialchars(xlt('You are not authorized to schedule appointments.'), ENT_NOQUOTES);
```

Comment:
hi @joshuaaguilar20 ,
xlt is basically shorthand for htmlspecialchars(xl('label')), so this should instead look like:
```
echo xlt('You are not authorized to schedule appointments.');
```